lib/pbench/test/unit/server/query_apis/commons.py

```diff
@@ -8,6 +8,7 @@
 from typing import AnyStr, Type
 
 from pbench.server.api.resources import JSON, ParamType
+from pbench.server.database.models.datasets import Dataset, Metadata
```

Comment:
Out of order!

Reply:
(I.e., you need to swap lines 11 and 12.... 😁)

Reply:
(And, @dbutenhof, don't look too closely at `lib/pbench/server/api/__init__.py`.... 🤣)